include/matx/core/sparse_tensor_format.h

```diff
@@ -165,23 +166,24 @@ template <int D, typename... S> class SparseTensorFormat {
 
   static constexpr bool isCSC() {
     if constexpr (LVL == 2) {
-      using type0 = std::tuple_element_t<0, LvlSpecs>;
-      using type1 = std::tuple_element_t<1, LvlSpecs>;
+      using type0 = cuda::std::tuple_element_t<0, LvlSpecs>;
+      using type1 = cuda::std::tuple_element_t<1, LvlSpecs>;
       return type0::Expr::isId(1) && type0::Type::isDense() &&
              type1::Expr::isId(0) && type1::Type::isCompressed();
     }
     return false;
   }
 
-  template <typename CRD, int L = 0>
+  template <class CRD, int L = 0>
```

Comment:
We typically only use `typename` in here.

Reply:
Fair enough. I will change that part back.

Reply:
Those are likely mistakes. I'll merge this for now and clean up later.